cypress/integration/ui_spec.js

```diff
@@ -0,0 +1,685 @@
+
+						cy.visit("/", {
+							onBeforeLoad: (win) => {
+								// eslint-disable-next-line no-param-reassign
+								win.parleySettings = parleyConfig;
+							},
+						});
+
+						cy.get("[id=app]").as("app");
+
+						clickOnLauncher();
+
+						cy.get("@app")
+							.find("[class*=announcement__]")
+							.first()
+							.should("have.text", welcomeMessage);
+					});
+				});
+				describe("placeholderMessenger", () => {
+					it("should change the input's placeholder text", () => {
+						const parleyConfig = {runOptions: {interfaceTexts: {placeholderMessenger: "This is the placeholder"}}};
+
+						cy.visit("/", {
+							onBeforeLoad: (win) => {
+								// eslint-disable-next-line no-param-reassign
+								win.parleySettings = parleyConfig;
+							},
+						});
+
+						cy.get("[id=app]").as("app");
+
+						clickOnLauncher();
+
+						cy.get("@app")
+							.find("[class^=text__]")
+							.find("textarea")
+							.should("have.attr", "placeholder", parleyConfig.runOptions.interfaceTexts.placeholderMessenger);
+
+						// Test if it changes during runtime
+						const newPlaceholder = "This is the placeholder #2";
+						cy.window().then((win) => {
+							// eslint-disable-next-line no-param-reassign
+							win.parleySettings.runOptions.interfaceTexts.placeholderMessenger = newPlaceholder;
+						});
+
+						cy.get("@app")
+							.find("[class^=text__]")
+							.find("textarea")
+							.should("have.attr", "placeholder", newPlaceholder);
+					});
+				});
+			});
+			describe("country", () => {
+				it("should change the language of interface texts", () => {
+					const parleyConfig = {
+						runOptions: {
+							country: "en",
+							interfaceTexts: {desc: "Messenger - EN"},
+						},
+					};
+
+					cy.visit("/", {
+						onBeforeLoad: (window) => {
+							// eslint-disable-next-line no-param-reassign
+							window.parleySettings = parleyConfig;
+						},
+					});
+
+					cy.get("[id=app]").as("app");
+
+					clickOnLauncher();
+
+					cy.get("@app")
+						.find("[class^=text__]")
+						.find("textarea")
+						.should("have.attr", "placeholder", InterfaceTexts.english.inputPlaceholder);
+
+					// Test if it changes during runtime
+					cy.window().then((win) => {
+						// eslint-disable-next-line no-param-reassign
+						win.parleySettings.runOptions.country = "nl";
+					});
+
+					cy.get("@app")
+						.find("[class^=text__]")
+						.find("textarea")
+						.should("have.attr", "placeholder", InterfaceTexts.dutch.inputPlaceholder);
+
+					// Extra test to validate that custom interface texts (desc we set above)
+					// have not been overwritten by the new language's defaults
+					cy.get("@app")
+						.find("[class^=title__]")
+						.should("have.text", parleyConfig.runOptions.interfaceTexts.desc);
+				});
+			});
+		});
+		describe("roomNumber", () => {
+			it("should register a new device when switching accounts", () => {
+				const parleyConfig = {roomNumber: "0W4qcE5aXoKq9OzvHxj2"};
```

Comment:
Moet `"0W4qcE5aXoKq9OzvHxj2"` niet ergens in een algemenere constant

Reply:
Is al gedaan in 8a2b185b61d94cf80e7f542b3b1b1ce8df0e81c6